notebooks/ltx-video/ltx-video.ipynb

```diff
@@ -0,0 +1,678 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    %pip install -qU "torch&gt;=2.1.0" "diffusers&gt;=0.28.2" "transformers&gt;=4.44.2" "sentencepiece&gt;=0.1.96" "huggingface-hub~=0.25.2" "einops" "accelerate" "matplotlib" "imageio[ffmpeg]" "nncf&gt;=2.14" --extra-index-url https://download.pytorch.org/whl/cpu
</pre><p>could you please specify torchvision explicitly to prevent error like we see now in docker validation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2686/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Added</p>